concent-vm/configure.yml

```diff
@@ -60,6 +60,14 @@
               - python3-psutil
               - rustc
 
+        # FIXME: Why doesn't the task above install rustc from testing? Find the cause and fix it properly.
+        # Reported as https://github.com/golemfactory/concent-deployment/issues/319
+        - name:   Quick and dirty workaround for rustc not getting installed from testing
+          shell:  apt-get install rustc
```

Comment:
Why you are not using backslash here ?

Reply:
Because it's not needed.

See this about line breaking in YAML: [Understanding multi line strings in YAML and Ansible (Part I - YAML)](https://adminswerk.de/multi-line-string-yaml-ansible-I/)